storybook/pages/quicklink.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
I'm not sure if quicklink is needed. It could be a modifier of link if I understand correctly.